experimental/dds/tree2/src/feature-libraries/typed-schema/flexList.ts

```diff
@@ -3,39 +3,79 @@
  * Licensed under the MIT License.
  */
 
+/** @remarks Used by `markEager`and `isMarkedEager`. */
```

Comment:
Can we document the semantics of this rather than just list (without links) the places that use it.

If I'm writing code and I want to know if I can use this, these docs arn't useful.